docs/docs/schema-relations.md

```diff
@@ -100,7 +100,7 @@ Almost identical! But now there's an `id` and the SDL/scaffold generators will w
 ## Troubleshooting Generators
 
 Are you getting errors when generating SDLs or scaffolds for your Prisma models?
-There's a known issue in Redwood's GraphQL type generation that happens when generating SDL for or scaffolding out a Prisma model that has relations before the SDL for the related model exists.
+There's a known limitation in Redwood's GraphQL type generation that happens when generating SDL for, or scaffolding out, a Prisma model that has relations, before the SDL for the related model exists.
```

Comment:
Could we do without the last comma? It reads a little awkward. I'm open to italicizing or bolding the rest of the line if you were going for emphasis?
```suggestion
There's a known limitation in Redwood's GraphQL type generation that happens when generating SDL for, or scaffolding out, a Prisma model that has relations _before the SDL for the related model exists_.
```

Reply:
Yeah, you're right, it's better without that last comma. Thanks :)